### PR TITLE
ONE 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/venerated_rotpriest.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Venerated Rotpriest
+ManaCost:G
+Types:Creature Phyrexian Druid
+PT:1/2
+K:Toxic:1
+T:Mode$ BecomesTarget | ValidTarget$ Creature.YouCtrl | ValidSource$ SpellAbility | TriggerZones$ Battlefield | Execute$ TrigPoison | TriggerDescription$ Whenever a creature you control becomes the target of a spell, target opponent gets a poison counter
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/venerated_rotpriest.txt:6]: `ValidSource$ Spell`

File: forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Mercurial Spelldancer
+ManaCost:1 U
+Types:Creature Phyrexian Rogue
+PT:2/1
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | Description$ CARDNAME can't be blocked.
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigDelayTrig | CombatDamage$ True | TriggerDescription$ Whenever Mercurial Spelldancer deals combat damage to a player, you may remove two oil counters from it. If you do, when you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt:8]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigDelayTrig:AB$ DelayedTrigger| Cost$ SubCounter<2/OIL> | Execute$ TrigReveal | AILogic$ SpellCopy | Execute$ EffTrigCopy | ThisTurn$ True | Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | SpellDescription$ When you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt:9]: why two Execute...?

File: forge-gui/res/cardsfolder/upcoming/urabrasks_forge.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Urabrask's Forge
+ManaCost:2 R
+Types:Artifact
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of combat on your turn, put an oil counter on Urabrask's Forge, then create an X/1 red Phyrexian Horror creature token with trample and haste, where X is the number of oil counters on Urabrask's Forge. Sacrifice that token at the beginning of the next end step.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/urabrasks_forge.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigDelayTrig:AB$ DelayedTrigger| Cost$ SubCounter<2/OIL> | AILogic$ SpellCopy | Execute$ EffTrigCopy | ThisTurn$ True | Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | SpellDescription$ When you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
+SVar:EffTrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | MayChooseTarget$ True
+SVar:HasAttackEffect:TRUE
+DeckHas:Ability$Counters
+DeckHints:Type$Instant|Sorcery
+Oracle:Mercurial Spelldancer can't be blocked.\nWhenever you cast a noncreature spell, put an oil conter on Mercurial Spelldancer.\nWhenever Mercurial Spelldancer deals combat damage to a player, you may remove two oil counters from it. If you do, when you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt:14]: conter

File: forge-gui/res/cardsfolder/upcoming/norns_wellspring.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Norn's Wellspring
+ManaCost:1 W
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigScry | TriggerDescription$ Whenever a creature you control dies, scry 1 and put an oil counter on CARDNAME.
+SVar:TrigScry:DB$ Scry | ScryNum$ 1 | SubAbility$ DBCounter
+SVar:DBCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Draw | Cost$ 1 T SubCounter<2/OIL> | SpellDescription$ Draw a card
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/norns_wellspring.txt:7]: `.` at end of SpellDesc

File: forge-gui/res/cardsfolder/upcoming/ovika_enigma_goliath.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Name:Ovika, Enigma Goliath
+ManaCost:5 U R
+Types:Legendary Creature Phyrexian Nightmare
+PT:6/6
+K:Flying
+K:Ward:3 PayLife<3>
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you cast a noncreature spell, create X 1/1 red Phyrexian Goblin creature tokens, where X is the mana value of that spell. They gain haste until end of turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ r_1_1_phyrexian_goblin | TokenOwner$ You | PumpKeywords$ Haste | PumpDuration$ EOT
+SVar:X:TriggeredStackInstance$CardManaCostLKI
+DeckHas:Ability$Token & Type$Goblin
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ovika_enigma_goliath.txt:10]: probably add `SVar:BuffedBy:Card.nonCreature`

File: forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigDelayTrig | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you may remove two oil counters from it. If you do, when you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
+SVar:TrigDelayTrig:AB$ DelayedTrigger| Cost$ SubCounter<2/OIL> | AILogic$ SpellCopy | Execute$ EffTrigCopy | ThisTurn$ True | Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | SpellDescription$ When you cast your next instant or sorcery spell this turn, copy that spell. You may choose new targets for the copy.
+SVar:EffTrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | MayChooseTarget$ True
+SVar:HasAttackEffect:TRUE
+DeckHas:Ability$Counters
+DeckHints:Type$Instant|Sorcery
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/mercurial_spelldancer.txt:13]: probably add `SVar:BuffedBy:Card.nonCreature`

File: forge-gui/res/cardsfolder/upcoming/unctus_grand_metatect.txt [hunk 9]
```diff
@@ -0,0 +1,13 @@
+Name:Unctus, Grand Metatect
+ManaCost:1 U U
+Types:Legendary Artifact Creature Phyrexian Vedalken
+PT:2/4
+S:Mode$ Continuous | Affected$ Creature.Other+Blue+YouCtrl | AddTrigger$ LootTrig | Description$ Other blue creatures you control have "Whenever this creature becomes tapped, draw a card, then discard a card."
+SVar:LootTrig:Mode$ Taps | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME becomes tapped, draw a card, then discard a card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/unctus_grand_metatect.txt:6]: `TriggerDescription$ Whenever CARDNAME` should be `TriggerDescription$ Whenever this card`

File: forge-gui/res/cardsfolder/upcoming/venerated_rotpriest.txt [hunk 10]
```diff
@@ -0,0 +1,10 @@
+Name:Venerated Rotpriest
+ManaCost:G
+Types:Creature Phyrexian Druid
+PT:1/2
+K:Toxic:1
+T:Mode$ BecomesTarget | ValidTarget$ Creature.YouCtrl | ValidSource$ Spell | TriggerZones$ Battlefield | Execute$ TrigPoison | TriggerDescription$ Whenever a creature you control becomes the target of a spell, target opponent gets a poison counter
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/venerated_rotpriest.txt:6]: `.` at end of Desc – also Oracle

File: forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt [hunk 11]
```diff
@@ -0,0 +1,13 @@
+Name:Vindictive Flamestoker
+ManaCost:R
+Types:Creature Phyrexian Wizard
+PT:1/2
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Discard | Cost$ 6 R Sac<1/CARDNAME> | Defined$ You | Mode$ Hand | ReduceCost$ X | SubAbility$ DBDraw | SpellDescription$  Discard your hand. then draw four cards. This ability costs {1} less to activate for each oil counter on Vindictive Flamestoker.
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 4
+SVar:X:Count$CardCounters.OIL
+SVar:BuffedBy:nonCreature
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt:10]: Not sure if this should be `Card.nonCreature` – I see the version you have here on Deeproot Champion, but am unsure if it works that way

File: forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt [hunk 12]
```diff
@@ -0,0 +1,13 @@
+Name:Vindictive Flamestoker
+ManaCost:R
+Types:Creature Phyrexian Wizard
+PT:1/2
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Discard | Cost$ 6 R Sac<1/CARDNAME> | Defined$ You | Mode$ Hand | ReduceCost$ X | SubAbility$ DBDraw | SpellDescription$  Discard your hand. then draw four cards. This ability costs {1} less to activate for each oil counter on Vindictive Flamestoker.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt:7]: spacing and capitalization – capping problem also in Oracle

File: forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt [hunk 13]
```diff
@@ -0,0 +1,10 @@
+Name:White Sun's Twilight
+ManaCost:X W W
+Types:Sorcery
+A:SP$ GainLife | LifeAmount$ X | SubAbility$ DBToken | SpellDescription$ You gain X life. Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block." If X is 5 or more, destroy all other creatures. (Players dealt combat damage by a creature with toxic 1 also get a poison counter.)
+SVar:DBToken:DB$ Token | RememberTokens$ True | TokenAmount$ X | TokenScript$ c_1_1_a_phyrexian_mite_toxic_noblock | TokenOwner$ You | SubAbility$ DBWrath
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt:5]: are the mite tokens scripted?

File: forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt [hunk 14]
```diff
@@ -0,0 +1,10 @@
+Name:White Sun's Twilight
+ManaCost:X W W
+Types:Sorcery
+A:SP$ GainLife | LifeAmount$ X | SubAbility$ DBToken | SpellDescription$ You gain X life. Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block." If X is 5 or more, destroy all other creatures. (Players dealt combat damage by a creature with toxic 1 also get a poison counter.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt:4]: Some of this SpellDescription needs to be moved to other lines, or you need to make use of StackDescription stuff

Ensure that the stack description when this is cast doesn't have repeated or confusing lines

File: forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt (hunk 12 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt:7]: How is the stack description on this ability when activated?

File: forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt (hunk 11 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt:10]: Yea won't work
Please fix both

File: forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt (hunk 13 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt:5]: Mite token was done by #2213 also added to TypeList

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt]: Nice!

File: forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt [hunk 15]
```diff
@@ -2,8 +2,8 @@ Name:White Sun's Twilight
 ManaCost:X W W
 Types:Sorcery
 A:SP$ GainLife | LifeAmount$ X | SubAbility$ DBToken | SpellDescription$ You gain X life. Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block." If X is 5 or more, destroy all other creatures. (Players dealt combat damage by a creature with toxic 1 also get a poison counter.)
-SVar:DBToken:DB$ Token | RememberTokens$ True | TokenAmount$ X | TokenScript$ c_1_1_a_phyrexian_mite_toxic_noblock | TokenOwner$ You | SubAbility$ DBWrath 
-SVar:DBWrath:DB$ DestroyAll | ValidCards$ Creature.IsNotRemembered | ConditionCheckSVar$ X | ConditionSVarCompare$ GE5 | SubAbility$ DBCleanup
+SVar:DBToken:DB$ Token | RememberTokens$ True | TokenAmount$ X | TokenScript$ c_1_1_a_phyrexian_mite_toxic_noblock | TokenOwner$ You | SubAbility$ DBWrath | StackDescription$ Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt:5]: I think you could make lines 5 & 6 SpellDescription, then trim the SpellDescription in line 4
Less clutter that way – worth a try!

**Review by Northmoc — APPROVED** [review]
Nice work!

**Pull request merged**